packages/cli/src/lib/sync/syncQueries/index.ts

```
@@ -25,7 +25,7 @@ export default async function syncQueries({
     rootDir,
     APP_FOLDER,
     'static',
-    'latitude',
+    '.latitude',
```

Why is `.` now?

---

All files inside `views` that are not `index.html` will be copied to the `static` directory in the app folder. The only exception are hidden files and directories (starting with a dot).

We were using a `static/latitude` folder for the queries, but since it can now conflict with the user's view files, I changed it to `static/.latitude`, since hidden directories are not copied.